src/pymatgen/core/interface.py

```diff
@@ -2784,10 +2783,16 @@ def from_slabs(
             substrate_slab = substrate_slab.get_orthogonal_c_slab()
         if isinstance(film_slab, Slab):
             film_slab = film_slab.get_orthogonal_c_slab()
-        assert_allclose(film_slab.lattice.alpha, 90, 0.1)
```

Comment:
`math.isclose` is much faster (~ 500x) for comparing scalar than `assert_allclose`.


```python
import math
from numpy.testing import assert_allclose
import timeit


def run_tests():
    a = 0.123456789
    b = 0.123456789

    rtol = 1e-5
    atol = 1e-8

    assert math.isclose(a, b, rel_tol=rtol, abs_tol=atol)
    assert_allclose(a, b, rtol=rtol, atol=atol)

    # Measure performance in milliseconds
    num_runs = 1000000
    isclose_time = timeit.timeit(lambda: math.isclose(a, b, rel_tol=rtol, abs_tol=atol), number=num_runs) * 1000
    assert_allclose_time = timeit.timeit(lambda: assert_allclose(a, b, rtol=rtol, atol=atol), number=num_runs) * 1000

    print(f"\nPerformance results over {num_runs} runs with single float values:")
    print(f"math.isclose time: {isclose_time:.3f} ms")
    print(f"assert_allclose time: {assert_allclose_time:.3f} ms")
    print(f"Speedup: {assert_allclose_time / isclose_time:.2f}x")

run_tests()
```

Gives:
```
Performance results over 1000000 runs with single float values:
math.isclose time: 37.630 ms
assert_allclose time: 19495.991 ms
Speedup: 518.09x
```